src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -450,6 +490,10 @@ def report_metrics_for_detectors(
         "sdk_name": sdk_name,
         "is_early_adopter": organization.flags.early_adopter.is_set,
     }
+
+    if is_dry_run:
+        detected_tags["is_dry_run"] = True
```

Comment:
We should be able to filter by this on datadog just how we filter by is_early_adopter right?

Reply:
Addresssed.